dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/bnd.bnd

```diff
@@ -118,8 +124,12 @@ instrument.classesExcludes: \
   org/eclipse/persistence/internal/helper/*.class
 
 -buildpath: \
-    org.eclipse.persistence.core;version=2.7.7.v20200504-69f2c2b80d;strategy=exact,\
-    org.eclipse.persistence.jpa;version=2.7.7.v20200504-69f2c2b80d;strategy=exact,\
+    org.eclipse.persistence:org.eclipse.persistence.antlr;version=${eclFullVersion};strategy=exact,\
+    org.eclipse.persistence:org.eclipse.persistence.asm;version=${eclFullVersion};strategy=exact,\
+    org.eclipse.persistence:org.eclipse.persistence.core;version=${eclFullVersion};strategy=exact,\
```

Comment:
I assume you added these to the buildpath due to issues you ran into with the above comments for this file.

Reply:
Correct me if I'm wrong, but the only reason to add to the `-buildpath` is if `com.ibm.websphere.appserver.thirdparty.eclipselink.2.7\src` needs the imports and compile dependencies, right?

Reply:
Removed the extra entries, they're not needed.